app/report/views.py

```diff
@@ -64,7 +64,7 @@ def get():
             requests_closed = len([r for r in active_requests if r[0] == request_status.CLOSED])
             requests_opened = len(active_requests) - requests_closed
             if not (current_user.is_anonymous or current_user.is_public):
-                if (current_user.is_agency and current_user.agency.ein == agency_ein) or current_user.is_super:
+                if (current_user.is_agency and current_user.is_agency_admin(agency_ein)) or current_user.is_super:
                     if current_user.is_agency_admin or current_user.is_super:
```

Comment:
The agency user dropdown is not displayed when logged in as an agency user (active and admin). `.is_agency_admin` is no longer a property in the class and returns a method, not boolean. Maybe this check can be removed altogether considering you have already checked for `is_agency_admin(agency_ein)` immediately above.